src/main/scala/com/gu/scanamo/request/ScanamoPutRequest.scala

```diff
@@ -0,0 +1,15 @@
+package com.gu.scanamo.request
+
+import com.amazonaws.services.dynamodbv2.model.AttributeValue
+
+case class ScanamoPutRequest(
```

Comment:
I was a bit loathe to move away from using the underlying request objects directly, but their lack of a copy constructor makes working with them really painful for the type of transformations needed for compound conditionals.